package.json

```diff
@@ -46,6 +46,8 @@
   "overrides": {
     "semver-regex": "3.1.4",
     "conventional-changelog": "3.1.25",
-    "trim-newlines": "3.0.1"
+    "trim-newlines": "3.0.1",
```

Comment:
why these 2 new dependencies ?

Reply:
these are the overrides, basically we are hard defining versions so these are not upgraded to other versions (other dependencies must be dependent on these which would be using older versions of these dependencies) which have vulnerabilities.

Reply:
+1
Here we are overriding the dependencies required by other dependencies.